packages/avatar-group/src/vaadin-avatar-group.js

```diff
@@ -86,6 +88,7 @@ class AvatarGroup extends AvatarGroupMixin(ElementMixin(ThemableMixin(PolylitMix
         .opened="${this._opened}"
         .positionTarget="${this._overflow}"
         no-vertical-overlap
+        exportparts="overlay, content"
```

Comment:
The `backdrop` part is always hidden so IMO we shouldn't export it. The only reason it actually exists in the DOM is because of the dummy listener added in `OverlayMixin` (which could probably be removed).